src/components/game/components/board/Board.js

```diff
@@ -91,19 +69,21 @@ const Board = ({
       </div>
 
       <div className="buttons-row">
-        <div>{renderBuildDeckButton()}</div>
+      <Buttons id={"Play"} onClick={onClickPlayHandler} visible={phase >=1 ? false : true}/>
```

Comment:
```suggestion
      {phase >=1 && <Buttons id={"Play"} onClick={onClickPlayHandler} />}
```
Follow this example on other Button components as well and remove the visible prop